src/Native/Windows/build-native.cmd

```diff
@@ -112,7 +116,7 @@ exit /b 1
 
 :GenVSSolution
 :: Generate Version file
-msbuild /t:GenerateVersionHeader build.proj /p:NativeVersionHeaderFile="%__binDir%\obj\_version.h" /p:GenerateVersionHeader=true
+msbuild /t:GenerateVersionHeader build.proj /p:NativeVersionHeaderFile="%__binDir%\obj\_version.h" /p:GenerateVersionHeader=true %OfficialBuildIdArg%
```

Comment:
Could we be consistent in where this properties are being passed? I don't know if it is possible but if we are passing them in build.sh we should pass them in build.cmd too for consistency